src/error.rs

```diff
@@ -213,6 +242,30 @@ impl Error {
     }
 }
 
+impl From<actix_http::error::Error> for Error {
+    fn from(error: actix_http::error::Error) -> Error {
+        Error::new(ErrorKind::ActixHttpError(error.to_string()), None)
+    }
+}
+
+impl From<serde_json::error::Error> for Error {
+    fn from(error: serde_json::error::Error) -> Error {
+        Error::new(ErrorKind::SerdeJsonError(error.to_string()), None)
+    }
+}
+
+impl From<r2d2::Error> for Error {
+    fn from(error: r2d2::Error) -> Error {
+        Error::new(ErrorKind::R2D2Error(error.to_string()), None)
+    }
+}
+
+impl From<String> for Error {
+    fn from(error: String) -> Error {
+        Error::new(ErrorKind::GenericError(error), None)
+    }
+}
+
```

Comment:
We should be consistent.  For some errors above, we're creating an ErrorKind variant that takes the originating / source error.  In these cases, it looks like we're pulling out strings from errors and just carrying those.  In so doing, are we losing information, like enum variants in the original errors, that would allow for better programmatic handling of conditions?  Even if not, are the strings associated with those errors rich enough in their descriptions to make it easy to debug what's happening?  In some cases, noted above, we probably need better documentation about why these errors would occur in the context of Wrapgrapher.

Reply:
Wrapped the external errors inside new warpgrapher errors which describe the warpgrapher context. These are no longer used and have been removed.